cloud-consumption/src/main/resources/application.yml

```diff
@@ -108,6 +108,8 @@ vault:
       login.role: "cloudbreak.default"
 
 cb:
+  enabledplatforms: AWS,AZURE,YARN,GCP,MOCK
+  enabledgovplatforms: AWS
```

Comment:
would it makes sense to move these to the common module common-config.yml instead of addign to every service?